packages/shared/lib/core/nfts/utils/getIpfsHash.ts

```diff
@@ -0,0 +1,7 @@
+export function getIPFSHash(url?: string): string | undefined {
+    const ipfsPrefix = 'https://ipfs.io'
+
+    if (url?.includes(ipfsPrefix)) {
+        return url.slice(ipfsPrefix.length)
```

Comment:
Would it be better to use the `URL` module so we can address this comment and @marc2332 's comment? If we have `https://cloudflare-ipfs.com/ipfs/bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/Vincent_van_Gogh.html` and we want to get the `bafy...` part I think we could do this:

```js
const ipfsUrl = new URL("https://cloudflare-ipfs.com/ipfs/bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/Vincent_van_Gogh.html")
const path = ipfsUrl.pathname
return path.replace("/ipfs/", "").split("/")[0]
```